tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
```

Comment:
Why does one function need to be able to do both? Isn't one for tool code and the other for app/package code, so we pretty much always know which one we need?